docs/source/support/release-notes.rst

```diff
@@ -20,6 +20,17 @@ Navigator
 - Fixed a regression where Navigator console was not able to inspect contracts and events.
   See `#1454 <https://github.com/digital-asset/daml/issues/1454>`__.
 
+DAML Integration Kit
+~~~~~~~~~~~~~~~~~~~~
+
+- The :doc:`Ledger API Test Tool </tools/ledger-api-test-tool/index>` now also runs ``TransactionServiceIT`` as part of the conformance tests.
+  This requires to load additional ``.dar`` files into the ledger under test. Please refer to the updated instructions in the documentation.
```

Comment:
I think this is what you're saying, reject if incorrect
```suggestion
  This means you need to load additional ``.dar`` files into the ledger under test. Please refer to the updated instructions in the :doc:`documentation </daml-integration-kit/index>`.
```

Reply:
Thanks, @bethaitman.